src/common/dsp/oscillators/WavetableOscillator.cpp

```diff
@@ -438,29 +445,9 @@ void WavetableOscillator::process_block(float pitch0, float drift, bool stereo,
         float shape = l_shape.v;
         float intpart;
         shape *= ((float)oscdata->wt.n_tables - 1.f + nointerp) * 0.99999f;
-        tableipol = modff(shape, &intpart);
+        tableipol = shape;
+        modff(shape, &intpart);
         tableid = limit_range((int)intpart, 0, (int)oscdata->wt.n_tables - 2 + nointerp);
-
-        if (tableid > last_tableid)
```

Comment:
Why was this code removed?

Reply:
This is the logic for the old morph interpolation. It has to be removed for the new one to work.

Reply:
In that case please comment it out instead of removing it, since this code will have to remain here in order for the option switch to work.

Reply:
> In that case please comment it out instead of removing it, since this code will have to remain here in order for the option switch to work.

ok, but you you told me the opposite about my last pr that I should just remove instead of commenting, since there is a git history :)